src/components/Tutorial/TutorialStepSecondThoughtEnter.tsx

```diff
@@ -13,7 +13,7 @@ const TutorialStepSecondThoughtEnter = () => {
   const ready = useSelector(state => {
     if (!state.cursor) return true
     const headCursorValue = headValue(state, state.cursor)
-    return headCursorValue?.length > 0
+    return headCursorValue && headCursorValue?.length > 0
```

Comment:
`headCursorValue?.length > 0` is not something we can compare,
so again, ugly

Reply:
That's fine.